kodilla-rps/src/main/java/Player.java

```diff
@@ -0,0 +1,39 @@
+public class Player {
+
+    private String name;
+    private int numberOfWins = 0;
+    private InputtingData inputtingData;
+
+    public Player(InputtingData inputtingData) {
+        this.inputtingData = inputtingData;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getNumberOfWins() {
+        return numberOfWins;
+    }
+
+    public void addWin() {
+        numberOfWins++;
+    }
+
+    public ActionType playerAction() {
+        Messages.showAskToPlayerChoice();
+        int userChoice = inputtingData.introductionNumber();
```

Comment:
what will happen if user provide input out of scope?

Reply:
done